journal/modernized/27-chapter-24.adoc

```diff
@@ -748,8 +749,8 @@ In the course of the time we were together at his house,
 the circumstance of the females coming so freely
 on board the Henry Freeling was mentioned,
 and the fears that I entertained on their account; but he said,
-'`Yours is called the "`Praying ship which is the reason of their venturing
-on board as they do.`' However pleasant and satisfactory it is to know the
+"`Yours is called the "`Praying Ship`" which is the reason of their venturing
```

Comment:
This seems like a quote within a quote, so I think we should use single quotes for the inner quote, like so:

```
"`Yours is called the '`Praying Ship`' which is the reason of their venturing
```